styles.css

```diff
@@ -126,3 +128,261 @@
 .main__button:hover {
   opacity: .5;
 }
+
+@media (max-width: 1919px) {
+  .wrapper {
+    width: 935px;
+  }
+
+  .header {
+    height: 34px;
+    margin-bottom: 120px;
+  }
+
+  .menu {
+    display: flex;
+  }
+
+  .menu__item {
+    margin-left: 24px;
+    font: 15px/51px roboto, helvetica, arial, sans-serif;
+  }
+
+  .menu__item_active::after {
+    display: block;
+    top: 35px;
+  }
+
+  .main {
+    margin-top: 0;
+    background: top right no-repeat url(img/frame1024.svg);
+    background-size: 55%;
+  }
+
+  .main__title {
+    margin: 0;
+    font: 50px/66px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 40px;
+    padding: 0;
+    font: 17px/20px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 41px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 1365px) {
+  .wrapper {
+    width: 93%;
+  }
+
+  .header {
+    height: 34px;
+    margin-bottom: 120px;
+  }
+
+  .menu {
+    display: flex;
+  }
+
+  .menu__item {
+    margin-left: 24px;
+    font: 15px/51px roboto, helvetica, arial, sans-serif;
+  }
+
+  .menu__item_active::after {
+    display: block;
+    top: 35px;
+  }
+
+  .main {
+    margin-top: 0;
+    background: top right no-repeat url(img/frame1024.svg);
+    background-size: 55%;
+  }
+
+  .main__title {
+    margin: 0;
+    font: 50px/66px pt_serifbold, arial, helvetica, sans-serif;
+  }
+
+  .main__subtitle {
+    margin-top: 40px;
+    padding: 0;
+    font: 17px/20px roboto, arial, helvetica, sans-serif;
+  }
+
+  .main__button {
+    margin-top: 41px;
+    padding: 13px 16px 15px 23px;
+    font: 14px/16px roboto, arial, helvetica, sans-serif;
+  }
+}
+
+@media (max-width: 800px) {
+  .wrapper {
+    margin: auto;
+    width: 80%;
+  }
+
+  .menu {
+    display: none;
+  }
+
+  .logo__name {
+      opacity: 0;
```

Comment:
Codacy found an issue: [Expected indentation of 4 spaces (indentation)](https://app.codacy.com/gh/WebHeroSchool/HTML_SNN/pullRequest?prid=6307906)